### PR TITLE
Add capping option to `Tube` function

### PR DESCRIPTION
**Pull request by beroda**
resolves https://github.com/pyvista/pyvista/issues/6864

### CODE REVIEW

File: tests/core/test_geometric_objects.py [hunk 1]
```diff
@@ -429,6 +429,12 @@ def test_tube():
         pv.Tube(pointa, (10, 1.0))
 
 
+@pytest.mark.parametrize('capping', [True, False])
+def test_tube_capping(capping: bool):
+    tube: pv.PolyData = pv.Tube(capping=capping).clean().triangulate()
```
Comment (beroda) [inline tests/core/test_geometric_objects.py:434]: Had to clean due to duplicated points at the cylinder end borders

Comment (tkoyama010) [reply tests/core/test_geometric_objects.py:434]:
```suggestion
    # Had to clean due to duplicated points at the cylinder end borders.
    tube: pv.PolyData = pv.Tube(capping=capping).clean().triangulate()
```

**Comment by tkoyama010** [comment]
This is awesome thanks!

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Comment by tkoyama010** [comment]
We will merge 24 hours after the Pull Request is opened out of consideration for other reviewers 🙂

**Review by tkoyama010 — APPROVED** [review]

**Pull request merged**